### PR TITLE
Terraform main workflow

### PR DESCRIPTION
**Pull request by YakMan101**
Finished constructing the pipeline for main workflow (Eventbridge -> Step Function -> [FN Lambda, DN Lambda] -> ECS Task)

Also created initial docker file for news_sentiment_analyser to aid in testing cloud pipeline.

Other small updates:
- updated terraform readme.md
- deleted tools/ folder
- deleted article combiner folder and corresponding workflow

### CODE REVIEW

File: terraform/.gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@
 terraform.tfvars
 terraform.tfstate
 .terraform.lock.hcl
-terraform.tfstate.backup
\ No newline at end of file
+terraform.tfstate.backup
+.terraform*
+*.pem
+*.pemkey
```
Comment (megansam5) [inline terraform/.gitignore:8]: essential .gitignore stuff, nice!

File: news_sentiment_analyser/dockerise.sh
Comment (indiah444) [inline news_sentiment_analyser/dockerise.sh:1]: Brilliant idea to create this shell script to automate dockerising files

**Review by megansam5 — APPROVED** [review]
LGTM

**Review by indiah444 — APPROVED** [review]
Great terraform files, really well documented and thought out, good idea to include the shell scripts for automating dockerising files!

**Pull request merged**